src/widget/friendwidget.h

```diff
@@ -47,7 +47,7 @@ class FriendWidget : public GenericChatroomWidget
 public slots:
     void onAvatarChange(int FriendId, const QPixmap& pic);
     void onAvatarRemoved(int FriendId);
-    void setAlias(const QString& alias);
+    void setAlias(const QString& alias) const;
```

Comment:
Ok, then the question is: If it has nothing to do with the surrounding class context, why does this slot even exist? :)